src/components/course/CourseReview.jsx

```diff
@@ -110,7 +110,7 @@ const CourseReview = () => {
             onClick={() => { setShowInfoContent(REVIEW_SECTION_CONTENT.DEMAND_AND_GROWTH); }}
             className={`number-color mb-0 ${ showInfoContent === REVIEW_SECTION_CONTENT.DEMAND_AND_GROWTH && 'text-underline'}`}
           >
-            {courseReviews.totalEnrollments}
+            {courseReviews?.totalEnrollments.toLocaleString('en-US')}
```

Comment:
[question] What if the user is using a locale that is not `en-US`? Would we want the number to be formatted according to their locale? If no locale is provided, I believe JavaScript detects and uses the current locale (e.g., if the app is being viewed in a non-US locale for i18n).